src/dev-app/datepicker/datepicker-demo.html

```diff
@@ -126,8 +126,8 @@ <h2>Input disabled, datepicker popup enabled</h2>
   <mat-datepicker-toggle [for]="datePicker3"></mat-datepicker-toggle>
   <mat-form-field>
     <mat-label>Input disabled, datepicker enabled</mat-label>
-    <input matInput disabled [matDatepicker]="datePicker3" [(ngModel)]="date" [min]="minDate"
-           [max]="maxDate" [matDatepickerFilter]="filterOdd ? dateFilter : undefined">
+    <input matInput [disabled]="true" [matDatepicker]="datePicker3" [(ngModel)]="date"
```

Comment:
Could also set it to just `disabled`.

Reply:
Unfortunately not, that's what we had before. We need to use a boolean so that `NgModel#disabled` is also happy.